common/traits/hfe_leader_trait.txt

```diff
@@ -1197,14 +1260,24 @@ hfe_generic_trait_noble = {
 	icon = gfx/interface/icons/other/hfe_noble.dds
 	leader_potential_add = {
 		hfe_trigger_leaders_traits_enabled = yes
+		exists = from
 		exists = species
-		species = {
-			hfe_trigger_is_robot = no
+		OR = {
+			species = {
+				hfe_trigger_is_robot = no
+			}
+			FROM = { has_country_flag = synthetic_empire }
 		}
-		exists = from
 		from = {
 			is_gestalt = no
-			has_authority = auth_imperial
+			OR = {
+				has_authority = auth_imperial
+				has_civic = civic_aristocratic_elite
+				has_civic = civic_feudal_realm
```

Comment:
i think civic_aristocratic_elite and civic_feudal_realm tied to auth_imperial

Reply:
Oligarchies can get some of these Civics as well.